app/models/vm_or_template/operations/snapshot.rb

```diff
@@ -42,6 +42,12 @@ module VmOrTemplate::Operations::Snapshot
         unsupported_reason_add(:revert_to_snapshot, unsupported_reason(:remove_snapshot))
       end
     end
+
+    virtual_column :supports_snapshot_memory, :type => :boolean
```

Comment:
Do we virtual column other supports things? I'm hesitant to expose it this way because then it's exposed to reporting.